src/components/ShellView/ResultTable.vue

```diff
@@ -40,20 +40,55 @@
             </th>
           </tr>
         </thead>
-        <tbody>
-          <tr v-for="(row, i) in rows" :key="i">
-            <td v-for="(cell, j) in row" :key="j">
-              <ul v-if="Array.isArray(cell)" class="list-group">
-                <li v-for="(item, k) in cell" :key="k" class="list-group-item">
-                  <b>{{ item.name }}:</b> {{ item.value }}
+      <tbody>
+      <tr v-for="(row, i) in rows" :key="i">
+        <td v-for="(cell, j) in row" :key="j">
+          <div v-if="Array.isArray(cell)" class="list-container">
+            <ul class="list-group">
+              <li v-for="(item, k) in cell" :key="k" class="list-group-item">
+                <b>{{ item.name }}:</b> {{ item.value }}
+              </li>
+            </ul>
+          </div>
+          <div v-else-if="cell.hasOwnProperty('_nodes') && cell.hasOwnProperty('_rels')" class="data-container">
+            <div class="flex-item">
+              <ul class="list-group">
+                <li v-for="(item, k) in cell._nodes" :key="k" class="list-group-item-o">
+                  <ul class="list-group">
+                    <li v-for="(field, k) in item" :key="k" class="list-group-item">
+                      <span v-if="field.name === 'LABEL'">
+                        <b>{{ field.value }}</b>
+                      </span>
+                      <span v-else>
+                        <b>{{ field.name }}:</b> {{ field.value }}
+                      </span>
+                    </li>
+                  </ul>
                 </li>
               </ul>
-
-              <span v-else>{{ cell }}</span>
-            </td>
-          </tr>
-        </tbody>
-      </table>
+            </div>
+            <div class="flex-item">
+              <ul class="list-group">
+                <li v-for="(item, k) in cell._rels" :key="k" class="list-group-item-o">
```

Comment:
What does `list-group-item-o` mean? You should review your CSS class naming. In general we use [bem naming convention](https://getbem.com/naming/)